packages/block-editor/src/components/page-template-picker/button.native.js

```diff
@@ -1,20 +1,47 @@
 /**
  * External dependencies
  */
-import { Button } from 'react-native';
+import { TouchableOpacity, Text, View } from 'react-native';
 
-const PickerButton = ( props ) => {
-	const {
-		label,
-		onPress,
-	} = props;
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	Icon,
+} from '@wordpress/components';
+import { withPreferredColorScheme } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import styles from './styles.scss';
+
+const PickerButton = ( {
+	icon,
+	label,
+	onPress,
+	getStylesFromColorScheme,
+} ) => {
+	const butonStyles = getStylesFromColorScheme( styles.button, styles.buttonDark );
+	const butonTextStyles = getStylesFromColorScheme( styles.buttonText, styles.buttonTextDark );
+	const accessibilityLabel = sprintf(
+		/* translators: accessibility text. Inform about list of predefined layout options. %1$s: Layout name, e.g About. */
+		__( 'Predefined layout picker. %1$s' ),
```

Comment:
After reading this https://make.wordpress.org/design/2019/11/14/blocks-patterns-and-layouts/ I used `Layout` instead of `Page template`.

Reply:
Nice, I wonder if we should start renaming things more broadly (file and component names)

Reply:
This seems like too much text before the actual button label. I'm not sure I would specify an `accessibilityLabel` here. Instead, we can add an `accessibilityHint="Double tap to select layout"`.

Ideally we should find a way to convey that this is a list of possible layouts rather than individual buttons, but I don't see any support for [accessibility containers](https://developer.apple.com/documentation/uikit/accessibility/uiaccessibilitycontainer) in React Native yet. Since we only have a couple buttons now, it's not a big deal, but something to keep in mind for the future.

Reply:
Good idea! I'll use `accessibilityHint` instead. I hope they add support for that, it'd be super great.

Reply:
Updated https://github.com/WordPress/gutenberg/pull/19307/commits/4c1c32684840fe0585d7203cd119c08fa13f6db9